### PR TITLE
HKG: Car Port for Hyundai Custin 1st Gen

### PR DESCRIPTION
**Pull request by Saber422**
Hyundai Custin is asia only car ,named Custo in China. Hyundai K , talk with Sotana dbc CAN BUS
test route"0bbe367c98fa1538|2023-08-19--01-58-42--0"

It's my fisrt time to PR a port car. I might be missing something. Waiting your instruction to complete it.

### CODE REVIEW

File: selfdrive/car/hyundai/values.py [hunk 1]
```diff
@@ -207,7 +208,8 @@ def init_make(self, CP: car.CarParams):
   ],
   CAR.TUCSON_HYBRID_4TH_GEN: HyundaiCarInfo("Hyundai Tucson Hybrid 2022-23", "All", car_parts=CarParts.common([CarHarness.hyundai_n])),
   CAR.SANTA_CRUZ_1ST_GEN: HyundaiCarInfo("Hyundai Santa Cruz 2022-23", car_parts=CarParts.common([CarHarness.hyundai_n])),
-
+  CAR.CUSTIN: HyundaiCarInfo("Hyundai Custin 2023", car_parts=CarParts.common([CarHarness.hyundai_k])), #Asia only, Named Custo in China
+0
```
Comment (sshane) [inline selfdrive/car/hyundai/values.py:212]:
```suggestion

```

Comment (sshane) [inline selfdrive/car/hyundai/values.py:212]:
```suggestion
  CAR.CUSTIN: HyundaiCarInfo("Hyundai Custin 2023", car_parts=CarParts.common([CarHarness.hyundai_k])), 
 # Asia only, Named Custo in China

```

**Comment by sshane** [comment]
Make sure to add to interface.py, and any applicable lists below in values.py You also need to add a public test route with rlogs uploaded to https://github.com/commaai/openpilot/blob/master/selfdrive/car/tests/routes.py

**Comment by sunnyhaibin** [comment]
Requested a test route with this PR branch. The new route will be updated once provided.

**Comment by Saber422** [comment]
@sshane finally got time to meet the owner.
thanks for @sunnyhaibin's help and I add a new route of CUSTIN. "0bbe367c98fa1538|2023-09-16--00-16-49--0"
There is also an experimental mode route of CUSTIN."0bbe367c98fa1538|2023-09-16--00-27-28--0"

**Comment by sunnyhaibin** [comment]
@Saber422 @sshane  This car port is ready to mark for review.

File: RELEASES.md [hunk 2]
```diff
@@ -6,6 +6,7 @@ Version 0.9.5 (2023-09-27)
 * New driving model
   * Improved navigate on openpilot performance using navigation instructions as an additional model input
 * Hyundai Azera 2022 support thanks to sunnyhaibin!
+* Hyundai Custin (Taiwan only) 2023 support thanks to sunnyhaibin and Saber422!
```
Comment (sshane) [inline RELEASES.md:9]: Fine to remove `Taiwan only` if it's not sold in the US https://github.com/commaai/openpilot/pull/29876#discussion_r1323871685

File: selfdrive/car/hyundai/interface.py [hunk 3]
```diff
@@ -141,6 +141,10 @@ def _get_params(ret, candidate, fingerprint, car_fw, experimental_long, docs):
       ret.wheelbase = 3.000
       # steering ratio according to Hyundai News https://www.hyundainews.com/assets/documents/original/48035-2022SantaCruzProductGuideSpecsv2081521.pdf
       ret.steerRatio = 14.2
+    elif candidate == CAR.CUSTIN:
+      ret.mass = 1690.  # from https://www.hyundai-motor.com.tw/clicktobuy/custin#spec_0
+      ret.wheelbase = 3.055
+      ret.steerRatio = 17.07  # from learner
```
Comment (sshane) [inline selfdrive/car/hyundai/interface.py:147]:
```suggestion
      ret.steerRatio = 15.3  # from learner
```

File: selfdrive/car/hyundai/hyundaican.py [hunk 4]
```diff
@@ -38,7 +38,7 @@ def create_lkas11(packer, frame, car_fingerprint, apply_steer, steer_req,
                          CAR.ELANTRA_HEV_2021, CAR.SONATA_HYBRID, CAR.KONA_EV, CAR.KONA_HEV, CAR.KONA_EV_2022,
                          CAR.SANTA_FE_2022, CAR.KIA_K5_2021, CAR.IONIQ_HEV_2022, CAR.SANTA_FE_HEV_2022,
                          CAR.SANTA_FE_PHEV_2022, CAR.KIA_STINGER_2022, CAR.KIA_K5_HEV_2020, CAR.KIA_CEED,
-                         CAR.AZERA_6TH_GEN):
+                         CAR.AZERA_6TH_GEN, CAR.CUSTIN_1ST_GEN):
```
Comment (sshane) [inline selfdrive/car/hyundai/hyundaican.py:41]: What does the dash display without this?

**Review by sshane — APPROVED** [review]

File: selfdrive/car/hyundai/hyundaican.py (hunk 4 above)
Comment (sunnyhaibin) [reply selfdrive/car/hyundai/hyundaican.py:41]: @sshane  IIRC, I don't think it displays anything, but EPS would fault and does not allow steering.

**Pull request merged**